regolith/helpers/a_presentationhelper.py

```diff
@@ -145,7 +151,35 @@ def db_updater(self):
                     wait_bool = add_to_google_calendar(event)
                     jump += 1
 
+        # if gitlab box is checked, creates gitlab repo in talks group
+
+        if rc.gitlab:
+            headers = {
+                # TODO : store this in environment variable somehow
+                'PRIVATE-TOKEN': '<api_authentication_key>'
```

Comment:
The way we are doing this now is that each user has a `user.json` in `~/.config/regolith/`

These are typically lying unencrypted so we may want to revisit how we do this, but for now it is ok.